code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -448,6 +459,9 @@
 		return
 	if (!isflockmob(usr))
 		return
+	if (src.selected_by)
+		boutput(usr, "<span class='alert'>This drone is receiving a command!</span>")
+		return
```

Comment:
```suggestion
```
Probably fine to let someone mousedrop move a drone while selected?

Reply:
There could be cases where the person is mouse dropping the drone in a direction opposite of where you want it to go

Reply:
Eh, I'd say the case of mousedrop movement just not working because the drone is glowy is likely worse, but it's not too big of a deal.